infrastructure/quick-deploy/aws/parameters.tfvars

```diff
@@ -370,11 +405,13 @@ node_exporter = {
   node_selector = {}
 }
 
-windows_exporter = {
-  node_selector = {
-    "plateform" = "windows"
-  }
-}
+# Uncomment the following windows_exporter to deploy node exporter on windows node group
+#node exporter for windows
+# windows_exporter = {
+#   node_selector = {
+#     "platform" = "windows"
+#   }
+# }
```

Comment:
I think this could always be uncommented. It is not an error to have a daemon set without any node.